devicetypes/smartthings/aeon-home-energy-meter-c3.src/aeon-home-energy-meter-c3.groovy

```diff
@@ -94,11 +94,11 @@ def parse(String description) {
 	def cmd = zwave.parse(description, [0x31: 1, 0x32: 1, 0x60: 3])
 	if (cmd) {
 		result = createEvent(zwaveEvent(cmd))
+		log.debug "Parse returned ${result?.descriptionText}"
+        storeGraphData(result.name, result.value)
+	}else{
+		log.debug "Null zwave command"
```

Comment:
This is kind of ambiguous.  You should be more descriptive like "zwave.parse returned null command.  Cannot create event" because that's what was happening to cause you to end up in that flow control block. Also put a space before and after "else" clause.  It's standard to do it like } else {